lib/elixir/src/elixir_locals.erl

```diff
@@ -105,9 +105,15 @@ warn_unused_local(File, Module, All, Private) ->
 
 ensure_no_undefined_local(File, Module, All) ->
   if_tracker(Module, [], fun(Tracker) ->
-    [elixir_errors:form_error(Meta, File, ?MODULE, {Error, Tuple})
-     || {Meta, Tuple, Error} <- ?tracker:collect_undefined_locals(Tracker, All)],
-    ok
+    case ?tracker:collect_undefined_locals(Tracker, All) of
+      [] -> ok;
+
+      List ->
+        [{FirstMeta, _, _} | _] = Sorted = lists:sort(List),
```

Comment:
Why is the list sorted?

Reply:
Now that we show many, I am sorting by meta, which is the line at this point.

Reply:
👌

There are no tests checking the order, should there be?